mono/mini/mini-runtime.h

```diff
@@ -337,6 +337,22 @@ extern GList* mono_aot_paths;
 extern MonoDebugOptions mini_debug_options;
 extern GSList *mono_interp_only_classes;
 
+/*
+This struct describes what execution engine feature to use.
+The subsume and will eventually sunset mono_aot_only / mono_llvm_only and friends.
```

Comment:
"The subsume and will" -> "This subsume will"?

Reply:
Fixed